frontend/src/components/secure-message/MessageDisplay.vue

```diff
@@ -102,9 +102,10 @@
                   <v-icon v-else>mdi-email-open-outline</v-icon>
                   <span class="ml-1 markAsSpan">{{`Mark As ${secureExchange.isReadByMinistry ? 'Unread' : 'Read'}` }}</span>
                 </v-btn>
-                <v-btn id="claimAsButton" class="my-4 mx-2" v-on:click="clickClaimMsgButton" :disabled="!isEditable()">
-                  <v-icon>{{ secureExchange.reviewer ? 'mdi-account-off-outline' : 'mdi-account-check-outline' }}</v-icon>
-                  <span class="ml-1">{{ secureExchange.reviewer ? 'Unclaim' : 'Claim' }}</span>
+                <v-btn id="claimAsButton" class="my-4 mx-2" v-on:click="clickClaimMsgButton" :disabled="!isEditable() && !isClaimable() && !isUnClaimable()">
+                  <v-icon>{{ !isClaimable() ? 'mdi-account-off-outline' : 'mdi-account-check-outline' }}</v-icon>
+                  <span v-if="isClaimable()" class="ml-1">Claim</span>
+                  <span v-if="!isClaimable()" class="ml-1">Unclaim</span>
                 </v-btn>
```

Comment:
I think we can combine the spans on lines 107 and 108
 <span class="ml-1">{{ isClaimable() ? 'Claim' : 'Unclaim' }}</span>
 
 Unless there's some issue with inline conditional statements for vue text.

Reply:
Definitely, I don't know why i split them up.